server/game/cards/06-CotE/IaijutsuMaster.js

```diff
@@ -0,0 +1,25 @@
+const DrawCard = require('../../drawcard.js');
+
+class IaijutsuMaster extends DrawCard {
+    setupCardAbilities(ability) {
+        this.reaction({
+            title: 'Change your bid by 1 during a duel',
+            when: {
+                onHonorDialsRevealed: () => this.game.currentDuel && this.game.currentDuel.isInvolved(this.parent)
```

Comment:
You shouldn't use `this.parent` in when conditions.  Use `context.source.parent` instead.